shardingsphere-db-protocol/shardingsphere-db-protocol-mysql/src/main/java/org/apache/shardingsphere/db/protocol/mysql/packet/binlog/row/column/value/integer/MySQLBitBinlogProtocolValue.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.db.protocol.mysql.packet.binlog.row.column.value.integer;
+
+import org.apache.shardingsphere.db.protocol.mysql.packet.binlog.row.column.MySQLBinlogColumnDef;
+import org.apache.shardingsphere.db.protocol.mysql.packet.binlog.row.column.value.MySQLBinlogProtocolValue;
+import org.apache.shardingsphere.db.protocol.mysql.payload.MySQLPacketPayload;
+
+import java.io.Serializable;
+
+/**
+ * BIT type value of MySQL binlog protocol.
+ */
+public class MySQLBitBinlogProtocolValue implements MySQLBinlogProtocolValue {
+    
+    @Override
+    public Serializable read(final MySQLBinlogColumnDef columnDef, final MySQLPacketPayload payload) {
+        return payload.readLong(getLength(columnDef.getColumnMeta()));
+    }
+    
+    private int getLength(final int meta) {
+        int nbits = ((meta >> 8) * 8) + (meta & 0xff);
+        return (nbits + 7) / 8;
+    }
+}
```

Comment:
Can we consider about reuse with MySQLNullBitmap?

Reply:
I have discuss with Lucas. I think we can't resule MySQLNullBitmap, they are different.